tests/integration/local/start_api/lambda_authorizers/test_cfn_authorizer_definitions.py

```diff
@@ -0,0 +1,331 @@
+import pytest
+import requests
+from tests.integration.local.start_api.start_api_integ_base import (
+    StartApiIntegBaseClass,
+    WritableStartApiIntegBaseClass,
+)
+from parameterized import parameterized_class
+
+
+@parameterized_class(
+    ("template_path", "endpoint", "parameter_overrides"),
+    [
+        ("/testdata/start_api/lambda_authorizers/cfn-apigw-v1.yaml", "requestauthorizertoken", {}),
```

Comment:
Will this work with a string which starts with forward-slash? Are we prepending something else to these template locations?

Reply:
The template path is appended to the integration test folder with string concatenation in the [parent class](https://github.com/aws/aws-sam-cli/blob/ea78e6f61924bc0aa16bb70715952e1ab400f9c0/tests/integration/local/start_api/start_api_integ_base.py#L40). I haven't checked whether or not this behaviour is still consistent with forward slashes (so basically Windows). Will double check this behaviour on a Windows machine before merging.